cloud_blobstore/s3.py

```diff
@@ -193,6 +198,8 @@ def get(self, bucket: str, key: str) -> bytes:
             if ex.response['Error']['Code'] == "NoSuchKey":
                 raise BlobNotFoundError(ex)
             raise BlobStoreUnknownError(ex)
+        except (ConnectTimeout, ReadTimeout) as ex:
```

Comment:
This is such a common pattern that it may make sense to wrap each method with a decorator that encapsulates this behavior.

Downside is that the entire method is in a try-clause.

Something to consider.

Reply:
The timeout exceptions are very specific, so this seems reasonable.